### ISSUE TITLE
Problem with bintrayEnsurePackage task

### ISSUE THREAD

**Issue by laughedelic** [sbt/sbt-bintray#21]
Hi, I'm trying to use this plugin for publishing and get this error:

``` sbt
[error] (*:bintrayEnsurePackage) java.lang.NoSuchMethodError: 
    org.json4s.Implicits.string2jvalue(Ljava/lang/String;)Lorg/json4s/JsonAST$JString;
```

I tried sbt launcher `0.13.2` and `0.13.0`

The only similar thing that I found is https://github.com/wordnik/swagger-core/issues/431, so I tried to remove all other plugin dependencies in case if there are some conflicts — doesn't help.

**Comment by prettynatty** [comment]
I have the same issue.
My project doesn't have any plugins except bintray-sbt but I have ensime-sbt plugin enabled globally.

**Comment by prettynatty** [comment]
Disabling ensime-sbt doesn't help.

**Comment by velvia** [comment]
I have the same issue, reproducible on master branch of http://github.com/spark-jobserver/spark-jobserver.  Here is our plugins.sbt:
```scala
addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.7.4")

addSbtPlugin("io.spray" % "sbt-revolver" % "0.7.2")       // For quick restarts for web development

addSbtPlugin("com.typesafe.sbt" % "sbt-scalariform" % "1.3.0")

addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.11.2")

addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.6.0")

resolvers += Classpaths.sbtPluginReleases

addSbtPlugin("org.scoverage" %% "sbt-scoverage" % "1.0.4")

resolvers += Resolver.url(
  "bintray-sbt-plugin-releases",
    url("http://dl.bintray.com/content/sbt/sbt-plugin-releases"))(
        Resolver.ivyStylePatterns)

addSbtPlugin("me.lessis" % "bintray-sbt" % "0.1.1")

addSbtPlugin("me.lessis" % "ls-sbt" % "0.1.3")

addSbtPlugin("com.github.gseitz" % "sbt-release" % "0.8.5")

```

I'm not sure how to get dependencyGraph of the plugins, otherwise would find out what conflicts there are.

**Comment by velvia** [comment]
Oh, this was fixed for me by upgrading to 0.2.1.   There were issues with 0.3.0, which I filed #60 for.

**Comment by 2m** [comment]
Closing since reported to be fixed.
